src/Sulu/Bundle/TestBundle/Resources/dist/parameters.yml

```diff
@@ -0,0 +1,14 @@
+parameters:
+    database_mysql_host:     localhost
+    database_mysql_user:     root
+    database_mysql_password: ~
+    database_mysql_port:     ~
+
+    database_pgsql_host:     localhost
+    database_pgsql_user:     postgres
+    database_pgsql_password: ~
+    database_pgsql_port:     ~
```

Comment:
I think @carltongibson is right, might be convenient to have this setting here. We can leave it.